.github/workflows/biased-lang.yml

```diff
@@ -0,0 +1,16 @@
+name: Detecting Biased Language
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  biased_lang:
+    runs-on: ubuntu-latest
+    name: Detecting Biased Language
+    steps:
+      - uses: actions/checkout@v2
+      - id: pink-panther
+        uses: splunk/pink-panther@main
+        continue-on-error: true
```

Comment:
Won't this cause the workflow to be considered successful even if there is biased language? Don't we want it to fail?

Reply:
That seems fine as an interim step, particularly if it checks the entire repository and not just changed files.  Would give us notice and time to start addressing the issues.  Hard to know how it works, though, since the code the action actually uses doesn't appear to be in the action repo.

Reply:
Some repo's can contain a slew of biased terms which can take quite a while to replace, so we didn't want this job to fail the workflow. Instead, we've added an error annotation as shown in the attached photo to notify users there are biased terms in the repo, but allow the workflow to continue.

As for the code itself, this is hosted in our internal Gitlab instance where we test, build, and publish for use in both GitLab CI and GitHub Actions. The image is essentially a Python script utilizing RipGrep to search through the repo for a list of biased terms Splunk defines, and outputs a summary in JSON format which is visible in the job.

<img width="809" alt="Screen Shot 2021-05-10 at 11 10 23 AM" src="https://user-images.githubusercontent.com/15026021/117705082-73b36900-b180-11eb-83dd-49bc86ec738d.png">

Reply:
Completely understandable. Let me circle back with the team and see what we're able to share publicly. As for the biased terms we are searching for currently, the list is short (only 4 terms) which you can also view in the JSON output during the job, [shown here](https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/2494884207?check_suite_focus=true#step:3:69)